ICAFIX/scripts/computeRecleanFeatures.m

```diff
@@ -0,0 +1,285 @@
+        csf.cdata(csf.cdata~=0)=1;
+        DROPOUT=reshape(DROPOUT,size(DROPOUT,1)*size(DROPOUT,2)*size(DROPOUT,3),size(DROPOUT,4));
+        dropout.cdata=DROPOUT(std(VOLUME,[],2)>0,:);
+        SBREF=read_avw([SubjFolderlist '/MNINonLinear/Results/' fMRIName '/' fMRIName '_SBRef.nii.gz']);
+        SBREFOrig=SBREF;
+        SBREF=reshape(SBREF,size(SBREF,1)*size(SBREF,2)*size(SBREF,3),size(SBREF,4));
+        sbref.cdata=SBREF(std(VOLUME,[],2)>0,:);
+        noisy.cdata=(sum(Volume.cdata,2)./sbref.cdata)>prctile(sum(Volume.cdata,2)./sbref.cdata,87.5);
+        edge.cdata=wmparc.cdata==0;
+        Volume.cdata=Volume.cdata./std(Volume.cdata(:));
+        CIFTI.cdata=CIFTI.cdata./std(CIFTI.cdata(:));
+
+        NonGreyParcels=[4 5 14 15 24 31 43 44 63 72 213 221 2 41 77 78 79 85 100 109 155 156 157 158 159 160 161 162 192 219 223 251 252 253 254 255 703 3000 3001 3002 3003 3004 3005 3006 3007 3008 3009 3010 3011 3012 3013 3014 3015 3016 3017 3018 3019 3020 3021 3022 3023 3024 3025 3026 3027 3028 3029 3030 3031 3032 3033 3034 3035 3100 3101 3102 3103 3104 3105 3106 3107 3108 3109 3110 3111 3112 3113 3114 3115 3116 3117 3118 3119 3120 3121 3122 3123 3124 3125 3126 3127 3128 3129 3130 3131 3132 3133 3134 3135 3136 3137 3138 3139 3140 3141 3142 3143 3144 3145 3146 3147 3148 3149 3150 3151 3152 3153 3154 3155 3156 3157 3158 3159 3160 3161 3162 3163 3164 3165 3166 3167 3168 3169 3170 3171 3172 3173 3174 3175 3176 3177 3178 3179 3180 3181 4000 4001 4002 4003 4004 4005 4006 4007 4008 4009 4010 4011 4012 4013 4014 4015 4016 4017 4018 4019 4020 4021 4022 4023 4024 4025 4026 4027 4028 4029 4030 4031 4032 4033 4034 4035 4100 4101 4102 4103 4104 4105 4106 4107 4108 4109 4110 4111 4112 4113 4114 4115 4116 4117 4118 4119 4120 4121 4122 4123 4124 4125 4126 4127 4128 4129 4130 4131 4132 4133 4134 4135 4136 4137 4138 4139 4140 4141 4142 4143 4144 4145 4146 4147 4148 4149 4150 4151 4152 4153 4154 4155 4156 4157 4158 4159 4160 4161 4162 4163 4164 4165 4166 4167 4168 4169 4170 4171 4172 4173 4174 4175 4176 4177 4178 4179 4180 4181 5001 5002 13100 13101 13102 13103 13104 13105 13106 13107 13108 13109 13110 13111 13112 13113 13114 13115 13116 13117 13118 13119 13120 13121 13122 13123 13124 13125 13126 13127 13128 13129 13130 13131 13132 13133 13134 13135 13136 13137 13138 13139 13140 13141 13142 13143 13144 13145 13146 13147 13148 13149 13150 13151 13152 13153 13154 13155 13156 13157 13158 13159 13160 13161 13162 13163 13164 13165 13166 13167 13168 13169 13170 13171 13172 13173 13174 13175 14100 14101 14102 14103 14104 14105 14106 14107 14108 14109 14110 14111 14112 14113 14114 14115 14116 14117 14118 14119 14120 14121 14122 14123 14124 14125 14126 14127 14128 14129 14130 14131 14132 14133 14134 14135 14136 14137 14138 14139 14140 14141 14142 14143 14144 14145 14146 14147 14148 14149 14150 14151 14152 14153 14154 14155 14156 14157 14158 14159 14160 14161 14162 14163 14164 14165 14166 14167 14168 14169 14170 14171 14172 14173 14174 14175];
```

Comment:
Again, consider `-volume-label-import` with `-discard-others` in advance of this function, we already have some text files that do this kind of thing in `global/config`.